File: Exercicio1.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL,"Portuguese");
    int valor1, valor2, soma;

    printf("Escreva um valor: ");
    scanf("%d",&valor1);
    printf("\nEscreva outro valor: ");
    scanf ("%d",&valor2);

    soma = valor1 + valor2;
    printf("\n\nA soma dos dois valores é igual a %d\n",soma);
    return 0;
}

File: Exercicio1.cs
using System;

class Program {
  public static void Main (string[] args)
        {
            int valor1, valor2, soma;

            Console.WriteLine("Escreva um valor: ");
            valor1 = int.Parse(Console.ReadLine());
            Console.WriteLine("Escreva outro valor: ");
            valor2 = int.Parse(Console.ReadLine());

            soma = valor1 + valor2;
            Console.WriteLine("A soma dos dois valores eh igual a " + soma);
            
        }
    }